app/helpers/sessions_helper.rb

```diff
@@ -36,4 +36,18 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  def current_user? user
+    user && user == current_user
```

Comment:
chỗ check này ko cần thiết phải check tồn tại user đâu em, chỉ cần check xem nó có giống với current_user thôi
```suggestion
    user == current_user
```